bin/storm.py

```diff
@@ -229,7 +231,7 @@ def jar(jarfile, klass, *args):
         daemon=False,
         jvmopts=JAR_JVM_OPTS + ["-Dstorm.jar=" + jarfile])
 
-def sql(sql_file, topo_nam):
+def sql(sql_file, topo_name):
```

Comment:
minor nit: You may want to rename topo_name to topology_name. We can have JIRA to add a test which uses 'storm sql' command.